src/_bundle.es5.js

```diff
@@ -0,0 +1,4 @@
+import '../node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js';
+import './polyfills/index.js';
+export { default } from './index.js';
+
```

Comment:
All generated es5 output starts with this file. We include the polyfills (needed for IE support) and the `custom-elements-es5-adapter.js` from the WCJS library.

Reply:
adapter is injected via rollup `output.banner` configuration